src/calculateRentalCost.js

```diff
@@ -27,8 +27,21 @@
  * @return {number}
  */
 
+calculateRentalCost(7);
+
 function calculateRentalCost(days) {
-  // write code here
+  const dayPrice = 40;
+  const rentalCost = days * dayPrice;
+  const threeDayDiscount = 20;
+  const sevenDayDiscount = 50;
+
+  if (days >= 3 && days < 7) {
+    return rentalCost - threeDayDiscount;
+  } else if (days >= 7) {
+    return rentalCost - sevenDayDiscount;
+  } else {
+    return rentalCost;
+  }
```

Comment:
apply early return pattern here
```suggestion
  if (days >= 3 && days < 7) {
    return rentalCost - threeDayDiscount;
  } 
  
  if (days >= 7) {
    return rentalCost - sevenDayDiscount;
  } 
  
  return rentalCost;
```